v2_papers/2009/kind_histogram.json

```diff
@@ -0,0 +1,179 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Histogram-Based Traffic Anomaly Detection",
+    "authors": [
+      "Kind, Andreas",
+      "Stoecklin, Marc Ph.",
+      "Dimitropoulos, Xenofontas"
+    ],
+    "publication_name": "IEEE Transactions on Network Service Management",
+    "publication_type": "peer_reviewed_journal",
+    "year": 2009,
+    "organization_publishers": [
+      "ieee"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "article",
+      "volume": "6",
+      "issue": "2",
+      "pages": "110--121"
+    },
+    "access_open": false,
+    "curated_by": "Meghdouri, f.",
+    "curated_last_revision": "11-04-2018",
+    "curated_revision_number": 2
+  },
+  "data": {
+    "datasets": [
+      {
+        "name": "campus of IBM Research",
+        "types": [],
+        "generation_year": 2004,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured"
+      },
+      {
+        "name": "data center",
+        "types": [],
+        "generation_year": 2007,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "covered_period": "months"
+      },
+      {
+        "name": "DARPA IDeval",
+        "availability": "public",
+        "format": "packet",
+        "types": [],
+        "generation": "altered_captured",
+        "generation_year": 1999,
+        "covered_period": "weeks",
+        "details": [
+          "raw"
+        ],
+        "subsets": []
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": true,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "no",
+    "transformations": [
+      "_histograms",
+      "_metric_space",
+      "entropy"
+    ],
+    "final_data_format": "_histograms",
+    "feature_selections": [
+      {
+        "name": "constant-removal",
+        "type": "feature_reduction",
```

Comment:
This feature selection method seems to me like a ``filter``, and not a ``feature_reduction``.

Reply:
Filter feature selection always reduces dimensionality (unless the filter is the trivial identity).
``feature_reduction`` refers to methods that transform the space in such a way that the feature information is kept (or mostly kept), but in a space of smaller dimensionality. A good example of this is PCA, which transforms the space to a new space in which each dimension is a linear combination of the original features.
I do understand the confusion with the name though, if you have any suggestions for it let me know :)